mappings/net/minecraft/nbt/NbtHelper.mapping

```diff
@@ -1,17 +1,17 @@
 CLASS net/minecraft/class_2512 net/minecraft/nbt/NbtHelper
 	FIELD field_11582 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD method_10681 toBlockState (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2680;
-		ARG 0 tag
+		ARG 0 nbt
 	METHOD method_10682 withProperty (Lnet/minecraft/class_2688;Lnet/minecraft/class_2769;Ljava/lang/String;Lnet/minecraft/class_2487;Lnet/minecraft/class_2487;)Lnet/minecraft/class_2688;
 		ARG 0 state
 		ARG 1 property
 		ARG 2 key
-		ARG 3 propertiesTag
-		ARG 4 mainTag
+		ARG 3 propertiesNbt
+		ARG 4 mainNbt
```

Comment:
Specifically a tag.

Reply:
as opposed to what other type of nbt? But fair enough this is in the NbtHelper class so using tag is fine here I suppose

Reply:
Why not just `properties` `main`